lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -498,16 +808,171 @@ typedef SPIRVMap<dwarf::LocationAtom, SPIRVDebug::ExpressionOpCode>
+  add(dwarf::DW_OP_breg8,               SPIRVDebug::Breg8);
+  add(dwarf::DW_OP_breg9,               SPIRVDebug::Breg9);
+  add(dwarf::DW_OP_breg10,              SPIRVDebug::Breg10);
+  add(dwarf::DW_OP_breg11,              SPIRVDebug::Breg11);
+  add(dwarf::DW_OP_breg12,              SPIRVDebug::Breg12);
+  add(dwarf::DW_OP_breg13,              SPIRVDebug::Breg13);
+  add(dwarf::DW_OP_breg14,              SPIRVDebug::Breg14);
+  add(dwarf::DW_OP_breg15,              SPIRVDebug::Breg15);
+  add(dwarf::DW_OP_breg16,              SPIRVDebug::Breg16);
+  add(dwarf::DW_OP_breg17,              SPIRVDebug::Breg17);
+  add(dwarf::DW_OP_breg18,              SPIRVDebug::Breg18);
+  add(dwarf::DW_OP_breg19,              SPIRVDebug::Breg19);
+  add(dwarf::DW_OP_breg20,              SPIRVDebug::Breg20);
+  add(dwarf::DW_OP_breg21,              SPIRVDebug::Breg21);
+  add(dwarf::DW_OP_breg22,              SPIRVDebug::Breg22);
+  add(dwarf::DW_OP_breg23,              SPIRVDebug::Breg23);
+  add(dwarf::DW_OP_breg24,              SPIRVDebug::Breg24);
+  add(dwarf::DW_OP_breg25,              SPIRVDebug::Breg25);
+  add(dwarf::DW_OP_breg26,              SPIRVDebug::Breg26);
+  add(dwarf::DW_OP_breg27,              SPIRVDebug::Breg27);
+  add(dwarf::DW_OP_breg28,              SPIRVDebug::Breg28);
+  add(dwarf::DW_OP_breg29,              SPIRVDebug::Breg29);
+  add(dwarf::DW_OP_breg30,              SPIRVDebug::Breg30);
+  add(dwarf::DW_OP_breg31,              SPIRVDebug::Breg31);
+  add(dwarf::DW_OP_regx,                SPIRVDebug::Regx);
+  add(dwarf::DW_OP_fbreg,               SPIRVDebug::Fbreg);
+  add(dwarf::DW_OP_bregx,               SPIRVDebug::Bregx);
+  add(dwarf::DW_OP_piece,               SPIRVDebug::Piece);
+  add(dwarf::DW_OP_deref_size,          SPIRVDebug::DerefSize );
+  add(dwarf::DW_OP_xderef_size,         SPIRVDebug::XderefSize );
+  add(dwarf::DW_OP_nop,                 SPIRVDebug::Nop);
+  add(dwarf::DW_OP_push_object_address, SPIRVDebug::PushObjectAddress );
+  add(dwarf::DW_OP_call2,               SPIRVDebug::Call2);
+  add(dwarf::DW_OP_call4,               SPIRVDebug::Call4);
+  add(dwarf::DW_OP_call_ref,            SPIRVDebug::CallRef);
+  add(dwarf::DW_OP_form_tls_address,    SPIRVDebug::FormTlsAddress);
+  add(dwarf::DW_OP_call_frame_cfa,      SPIRVDebug::CallFrameCfa);
+  add(dwarf::DW_OP_implicit_value,      SPIRVDebug::ImplicitValue);
+  add(dwarf::DW_OP_implicit_pointer,    SPIRVDebug::ImplicitPointer);
+  add(dwarf::DW_OP_addrx,               SPIRVDebug::Addrx);
+  add(dwarf::DW_OP_constx,              SPIRVDebug::Constx);
+  add(dwarf::DW_OP_entry_value,         SPIRVDebug::EntryValue);
+  add(dwarf::DW_OP_const_type,          SPIRVDebug::ConstTypeOp);
+  add(dwarf::DW_OP_regval_type,         SPIRVDebug::RegvalType);
+  add(dwarf::DW_OP_deref_type,          SPIRVDebug::DerefType);
+  add(dwarf::DW_OP_xderef_type,         SPIRVDebug::XderefType);
+  add(dwarf::DW_OP_reinterpret,         SPIRVDebug::Reinterpret);
```

Comment:
`dwarf::DW_OP_convert` is missing
https://github.com/llvm/llvm-project/blob/master/llvm/include/llvm/BinaryFormat/Dwarf.def#L664

Reply:
DW_OP_LLVM_convert is used instead.